Week2/homework/maartjes-work.js

```diff
@@ -45,19 +45,20 @@ const tuesday = [
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
-function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
-}
+const workDurationArray = maartjesTasks.map(key => key.duration);
```

Comment:
It would be nice to keep the existing `function computeEarnings(tasks, hourlyRate)` function here, and make it return what you have on line 54 (Maartje's un-rounded earnings).